server/src/test/java/io/crate/expression/InputFactoryTest.java

```diff
@@ -70,7 +71,7 @@ public class InputFactoryTest extends CrateDummyClusterServiceUnitTest {
             DataTypes.INTEGER.getTypeSignature(),
             DataTypes.INTEGER.getTypeSignature(),
             DataTypes.INTEGER.getTypeSignature()
-        ).withFeatures(Scalar.DETERMINISTIC_AND_COMPARISON_REPLACEMENT),
+        ).withFeatures(Sets.concat(Scalar.DETERMINISTIC_AND_COMPARISON_REPLACEMENT, Scalar.Feature.NULLABLE)),
```

Comment:
minor: `withFeature` does concat internally with previous Features. So you could write instead:
```
.withFeatures(Scalar.DETERMINISTIC_AND_COMPARISON_REPLACEMENT)
.withFeature(Scalar.Feature.NULLABLE)
```
It's also ok the way it is.

Reply:
Thanks! updated